client/src/components/ContentNavigator/ContentModel.ts

```diff
@@ -297,17 +307,14 @@ export class ContentModel {
         );
       }
 
-      const patchResponse = await this.connection.patch(
+      const patchResponse = await this.connection.put(
```

Comment:
Would you mind provide some reasoning for this change or add some comment to the source code?
It looks like `patch` is more efficient as `res.data` is not changed.

Reply:
Yeah, I updated this because `patch` was only available for certain content types. For dataflows, `patch` was not a valid request (but `put` was). So instead of having some logic to determine which to choose (`patch` or `put`), I just went with `put` since all types support that request